bom/README.md

```diff
@@ -0,0 +1,73 @@
+Guice Bill of Materials (BOM)
+============================
+This BOM contains all the Guice modules in a single dependency management.
+
+There are 2 ways of utilizing the BOM.
+
+1. Using it as the parent of your application's pom.
```

Comment:
Personally I'd discourage using the BOM as a parent pom for your application because of the way that pom inheritance works. I also suspect not many people would actually use the BOM in this way, most people would already be using an OSS or corporate pom (or want more control over the pom) in which case it's much easier to [import](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies) the BOM into your application pom.